core/src/main/java/org/n52/iceland/request/AbstractServiceRequest.java

```diff
@@ -175,6 +180,18 @@ public Locale getRequestedLocale() {
         return LocaleHelper.fromString(getRequestedLanguage());
     }
 
+    public String getOriginalRequest() {
```

Comment:
Well, if I do that, then we could get rid of hundreds of other ``has...`` and ``isSet...`` methods.

I like it because it hides that there is an Optional internally, but I'm open to changing this.